packages/oauth/oauth-scopes/src/scopes/blob-permission.ts

```diff
@@ -0,0 +1,106 @@
+import { Matchable } from '../lib/matchable.js'
```

Comment:
Any idea why these didn't appear as moved files?  Would appreciate any interesting call outs, since these permission classes largely look unchanged.

Reply:
The change is mostly aestetical. I added a `Matchable` (which I should probably rename to `ResourceScope` or similar) and moved the parsed logic to be a static prop. Nothing else changed.